### PR TITLE
Add unit tests to the backend package

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR adds unit tests for the dashboard backend.

### CODE REVIEW

File: packages/dashboard-backend/jest.config.js [hunk 1]
```diff
@@ -19,10 +19,11 @@ module.exports = {
...
-    'src/api/**/*.ts',
-    'src/services/**/*.ts',
-    'src/devworkspace-client/*.ts',
-    'src/devworkspace-client/services/**/*.ts',
+  coveragePathIgnorePatterns: [
+    'src/devworkspaceClient/__tests__/*',
   ],
+  testPathIgnorePatterns: [
+    'src/devworkspaceClient/__tests__/errors.spec.ts',
+    'src/devworkspaceClient/__tests__/integration.spec.ts',
```
Comment (olexii4) [inline packages/dashboard-backend/jest.config.js:27]: @akurinnoy I think we should remove  'src/devworkspaceClient/__tests__/errors.spec.ts' and
    'src/devworkspaceClient/__tests__/integration.spec.ts' at all.

File: packages/dashboard-backend/src/__tests__/app.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { FastifyInstance } from 'fastify';
+import { setup, teardown } from '../helpers/tests/appBuilder';
+
+describe('App', () => {
```
Comment (olexii4) [inline packages/dashboard-backend/src/__tests__/app.spec.ts:16]: I’m not sure I get what this test does.

Comment (akurinnoy) [reply packages/dashboard-backend/src/__tests__/app.spec.ts:16]: It tests that the app can start. I'll update the test to make that clear.

File: packages/dashboard-backend/src/devworkspaceClient/const/__tests__/models.spec.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { deletePolicy, deletionOptions } from '../models';
+
+test('models', () => {
```
Comment (olexii4) [inline packages/dashboard-backend/src/devworkspaceClient/const/__tests__/models.spec.ts:15]: I think we don't use those models in code anymore. It should be removed.

**Review by olexii4 — APPROVED** [review]

**Pull request merged**